sq/tests/dsm/extract_dsm_import_gpg.sh

```diff
@@ -124,9 +127,11 @@ $sq encrypt --signer-dsm-key="$bob_key_name" --recipient-cert "$alice_public" "$
 my_cat "$encrypted"
 
 comm "decrypt with (i) local key (ii) SDKMS key (iii) gpg-imported key"
-$gpg --output="$decrypted_gpg" --decrypt "$encrypted"
-diff "$message" "$decrypted_gpg"
 $sq decrypt --signer-cert="$bob_public" --recipient-key="$alice_local_priv" "$encrypted" --output "$decrypted_local"
 diff "$message" "$decrypted_local"
 $sq decrypt --signer-cert="$bob_public" --dsm-key="$alice_key_name" "$encrypted" --output "$decrypted_remote"
 diff "$message" "$decrypted_remote"
+$gpg --passphrase="test_passphrase" --output="$decrypted_gpg" --decrypt "$encrypted"
+diff "$message" "$decrypted_local"
```

Comment:
Is this supposed to be `$decrypted_gpg`? Or is `$decrypted_local` correct?

Reply:
Good catch, it got reverted/unmodified in a rebase (see https://github.com/fortanix/sq-dsm/issues/40)